src/abstractdataframe/join.jl

```diff
@@ -212,6 +212,7 @@ function update_row_maps!(left_table::AbstractDataFrame,
     return to_bimap(left_ixs), to_bimap(leftonly_ixs), to_bimap(right_ixs), to_bimap(rightonly_ixs)
 end
 
+
```

Comment:
Unrelated?

Reply:
not sure how that snuck in. Removed for next commit.